src/python/library/tritonclient/utils/__init__.py

```diff
@@ -269,3 +269,76 @@ def deserialize_bytes_tensor(encoded_tensor):
         offset += l
         strs.append(sb)
     return (np.array(strs, dtype=np.object_))
+
+
+def serialize_bf16_tensor(input_tensor):
+    """
+    Serializes a bfloat16 tensor into a flat numpy array of bytes.
+    The numpy array should use dtype of np.float32.
+
+    Parameters
+    ----------
+    input_tensor : np.array
+        The bfloat16 tensor to serialize.
+
+    Returns
+    -------
+    serialized_bf16_tensor : np.array
+        The 1-D numpy array of type uint8 containing the serialized bytes in 'C' order.
+
+    Raises
+    ------
+    InferenceServerException
+        If unable to serialize the given tensor.
+    """
+
+    if input_tensor.size == 0:
+        return np.empty([0], dtype=np.object_)
+
+    # If the input is a tensor of float32, then must flatten those into
+    # a 1-dimensional array containing the element bytes. All elements
+    # are concatenated together in "C" order.
```

Comment:
>in "C" order

change to 'in row-major form'

Edit: I see why you mention C order because of the `nditer()` call but I think row-major and column-major translates better than C and FORTRAN.

Reply:
Thanks. Clarified this for the bytes and bf16 functions in the latest commit.